frontend/src/app/featureFlags/FeatureFlagModal.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {
+  Button,
+  Checkbox,
+  Grid,
+  GridItem,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+} from '@patternfly/react-core';
+
+import { allFeatureFlags } from '~/concepts/areas/const';
+import { isFeatureFlag } from '~/concepts/areas/utils';
+import { FeatureFlagLauncherProps } from '~/app/featureFlags/FeatureFlagLauncher';
+
+type Props = FeatureFlagLauncherProps & { onClose: () => void };
+
+const FeatureFlagModal: React.FC<Props> = ({
+  dashboardConfig,
+  devFeatureFlags = {},
```

Comment:
Since you are setting it below, I think there is no need to set the value here? 🤔